pkg/virt-controller/services/template.go

```diff
@@ -370,6 +370,25 @@ func (t *templateService) RenderLaunchManifest(vmi *v1.VirtualMachineInstance) (
 			},
 		})
 	}
+
+        // Fix for kind "same host" migration, how to do it right ?
+        // keep in mind that the creation of the proudct_uuid i still ran manually
```

Comment:
for WORKER in kind-1.17.0-control-plane kind-1.17.0-worker; do
  docker exec $WORKER sh -c 'cat /proc/sys/kernel/random/uuid > /kind/product_uuid'
done

at the moment its not fully automatic, and the lines above are needed
if the file doesnt exists it would fail, again its WIP and known